src/topBar/drawer/Drawer.style.tsx

```diff
@@ -1,15 +1,20 @@
 import styled from 'styled-components'
 
-import { color, transition } from '../../_utils/branding'
+import { color, componentSizes, responsiveBreakpoints, transition } from '../../_utils/branding'
+
+export const StyledDrawer = styled.aside<{ zIndex?: number }>`
+  /* z-index handled in main application */
 
-export const StyledDrawer = styled.aside`
   &.kirk-drawer--open {
     position: fixed;
     top: 0;
     left: 0;
     bottom: 0;
     right: 0;
-    z-index: 1;
+    z-index: ${props =>
+      props.zIndex || 2}; /* z-index overridden in main application, above dimmer - below topBar */
```

Comment:
Shouldn't we use the `$zIndex` notation here?

Also I'm not sure we want to have the default to `2` here, we could simply remove the default no?

Reply:
Changed to use transient props (not used to it).

For the default, I thought I'd rather have defaults that work within the topBar component (as TopBar, Drawer, Dimmer are related) than making it a required prop. No real opinion.

Reply:
alright, made it a required prop (and also removed unnecessary style and added close() to the dimmer which is now included along with the drawer)